txscript/script.go

```diff
@@ -458,3 +458,15 @@ func GetPreciseSigOpCount(scriptSig, scriptPubKey []byte, bip16 bool) int {
 	shPops, _ := parseScript(shScript)
 	return getSigOpCount(shPops, true)
 }
+
+// IsUnspendable returns whether the passed scriptPubKey is unspendable, or
```

Comment:
You'll need to fix the reference to the variable name too when you update it.  This is why I don't generally like putting variable names in the comments.  It's better to simply describe what the function does.